src/Microsoft.DotNet.Interactive.Jupyter/Protocol/LanguageInfo.cs

```diff
@@ -71,7 +71,7 @@ public CSharpLanguageInfo(string version = "8.0") : base("C#", version, "text/x-
 
     public class FSharpLanguageInfo : LanguageInfo
     {
-        public FSharpLanguageInfo(string version = "4.5") : base("C#", version, "text/x-fsharp", ".fs", pygmentsLexer: "fsharp")
+        public FSharpLanguageInfo(string version = "4.5") : base("F#", version, "text/x-fsharp", ".fs", pygmentsLexer: "fsharp")
```

Comment:
not sure if these are the right version at this point in time

Reply:
Got it. I already commited a fix of them, just haven't pushed yet.